### DISCUSSION TITLE
export to CSV rounds decimal

### DISCUSSION THREAD

**Discussion by Tib76** [dbeaver/dbeaver#17766] [General]
Hi,

Please check the example attached
![line to export](https://user-images.githubusercontent.com/114601283/192824641-7cbc7838-d5f0-4619-8360-0169db4f151b.PNG)

when i export this line to a csv file i get 0 instead of 0.0000071 for the value column.
I just noticed also that when i copy paste the value column in a notepad for example it pastes 0 (??)

Can someone please help me with this ?
Thank you
Cheers,
Tibo

**Comment by LonwoLonwo** [comment]
Can you provide please your table DDL? With the value column. What database do you use?

![2022-09-29 22_45_37-DBeaver 22 2 1 - _localhost 1_ Script-20](https://user-images.githubusercontent.com/45152336/193127590-4299c799-2c61-4e2f-af51-a39aed4edcdd.png)

Additionaly can you provide your data format settings?

And did you try to change Export settings?

![2022-09-29 22_48_55-Data Transfer](https://user-images.githubusercontent.com/45152336/193128517-39b955de-7a6d-44af-be49-586001a45410.png)
